public/data/itsomsarraf_.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Om Sarraf",
+    "type": "personal",
+    "bio": "Open Source Enthusiast! Just getting started with it.",
+    "avatar": "https://github.com/itsomsarraf.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itsomsarraf",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itsomsarraf_",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```

Comment:
What do you think about capitalizing `i` here?
```suggestion
        "name": "Connect with me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question